db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java

```diff
@@ -163,7 +165,10 @@
 									"SYSSNAPSHOTS",
 									"SYSTOKENS",
                                     "SYSDUMMY1",
-									"SYSREPLICATION"
+									"SYSREPLICATION",
```

Comment:
Same here, nonCoreNames are for tables, not views as I understand, so the views should not be listed here, instead we should just put one entry for MON_GET_CONNECTION.

Reply:
Done.